tests/msvc-interface-test.sh

```diff
@@ -8,7 +8,6 @@ TESTNAME=$1
 
 script_location=$(dirname $0)
 test_dir_wsl=$($script_location/create-msvc-build-dir.sh)
-mkdir $test_dir_wsl 1>/dev/null 2>/dev/null || true
```

Comment:
Just confirming: is this removed because the `create-msvc-build-dir.sh` script guarantees the dir's creation?

Reply:
Yep.

The error message from this creating a directory that already exists made debugging other errors a little confusing.